Gemfile

```diff
@@ -7,7 +7,7 @@ gem 'rack-contrib', '~> 2.1.0'
 gem 'rails-observers', '~> 0.1.5'
 gem 'responders', '~> 3.0.1'
 gem 'mysql2', '~> 0.5.0'
-gem 'curb', '~> 0.9.4'
+gem 'curb', '~> 1.0', '>= 1.0.1'
```

Comment:
Let's just use: `'~> 1.0.1'`. I don't trust `curb` enough to bump minor versions with a `bundle update`. Eventually I would like to remove `curb` entirely in favor of either `faraday` or `http`.